frontend/src/app/components/AppView/AppView.tsx

```diff
@@ -179,7 +189,9 @@ function AppView(props: AppViewProps): ReactElement {
         />
         {/* Spacer fills up dead space to ensure the footer remains at the
         bottom of the page in larger views */}
-        {(!embedded || showFooter) && <StyledAppViewBlockSpacer />}
+        {(!embedded || showFooter) && (
```

Comment:
Maybe we only show the view block spacer and the app footer if `!containsChatInput`. This has two benefits: 1) we lower the bottom spacing between the chat input & the last message  2) the footer isn't visible in situations with horizontal scrolling.